test_six.py

```diff
@@ -792,14 +792,33 @@ def k():
     def f(g, assign, update):
         def w():
             return 42
-        w.glue = {"foo" : "bar"}
+        w.glue = {"foo": "bar"}
+        w.xyzzy = {"qux": "quux"}
```

Comment:
Why is this `xyzzy` attribute added?

Reply:
This was quite a while ago, so my memory is foggy about this.

I think this `xyzzy` here is to test that `wraps` does not fail if `wrapper` (the destination object) has the `xyzzy` attribute that it has to update, and `wrapped` (the original object) does not have it.